core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -574,6 +583,15 @@ object KafkaConfig {
   val PasswordEncoderKeyLengthProp =  "password.encoder.key.length"
   val PasswordEncoderIterationsProp =  "password.encoder.iterations"
 
+  /** ********* Raft Quorum Configuration *********/
+  val QuorumVotersProp = RaftConfig.QUORUM_VOTERS_CONFIG
```

Comment:
I know we have done so for other configs, but do we really need to duplicate these property strings?

Reply:
Probably not. And as you said, only to keep it consistent with the other props. I can go ahead and refer to the exported final string directly